src/js/tabs/adjustments.js

```diff
@@ -286,6 +286,14 @@ adjustments.cleanup = function (callback) {
     if (callback) callback();
 };
 
+$.fn.sortSelect = function() {
+    const op = this.children("option");
+    op.sort(function(a, b) {
```

Comment:
Maybe it's better to use a locale compare to be sure to order them correctly in any language? For example like here:

https://github.com/betaflight/betaflight-configurator/blob/bc6307adf74ddd557105d2dd8088267b206e9cbb/src/js/tabs/osd.js#L2644

Reply:
Right, I give it a try

Reply:
Maybe also make it into a common util function as we might need it in other places.